proto/terra/feeshare/v1beta1/genesis.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+package terra.feeshare.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "terra/feeshare/v1beta1/feeshare.proto";
+
+option go_package = "github.com/classic-terra/core/x/feeshare/types";
+
+// GenesisState defines the module's genesis state.
+message GenesisState {
+  // params are the feeshare module parameters
+  Params params = 1 [(gogoproto.nullable) = false];
+  // FeeShare is a slice of active registered contracts for fee distribution
+  repeated FeeShare fee_share = 2 [(gogoproto.nullable) = false];
+}
+
+// Params defines the feeshare module params
+message Params {
```

Comment:
Other x/modules seem to put this message contract in the top level  proto file. E.g. https://github.com/classic-terra/core/blob/649f7c4348bc437c6b26bbd9114febef84c7dd3a/proto/terra/market/v1beta1/market.proto#L9 && https://github.com/classic-terra/core/blob/649f7c4348bc437c6b26bbd9114febef84c7dd3a/proto/terra/oracle/v1beta1/oracle.proto#L9

Maybe we should mimic that convention? :)